src/main/java/Task.java

```diff
@@ -0,0 +1,13 @@
+public class Task {
+    protected String description;
+    protected boolean isComplete;
```

Comment:
Good job on naming the variables in the class with camelCase :)

Reply:
Hmmm... @Poopies99 , it would be better to say "good job on naming the boolean variable sound like boolean"